MdePkg/Include/Protocol/MmCommunication.h

```diff
@@ -26,7 +26,7 @@ typedef struct {
   ///
   /// Describes the size of Data (in bytes) and does not include the size of the header.
   ///
-  UINTN       MessageLength;
+  UINT64      MessageLength;

```

Comment:
Is this a bug or an update for a spec change?

Reply:
This was a bug fix for communication across architectures with different `UINTN` sizes. To introduce this in a backward compatible way, I'll close this PR and let a new PR add a `EFI_MM_COMMUNICATE_HEADER` structure with the change.